packages/cannon-worker-api/src/worker/index.ts

```diff
@@ -113,6 +113,14 @@ self.onmessage = ({ data }: { data: CannonMessage }) => {
     case 'setFixedRotation':
       state.bodies[data.uuid].fixedRotation = data.props
       break
+    case 'setFrictionGravity':
+      if (!data.props) {
+        // The user may reset frictionGravity.
+        delete state.world.frictionGravity
+        return;
```

Comment:
I'm not real keen on using return inside of this massive switch statement. Someone might add something after the switch and this single case will short circuit that.

Can we use a ternary statement?

```suggestion
state.world.frictionGravity = data.props ? new Vec3(...data.props) : data.props
```